src/petals/server/throughput.py

```diff
@@ -88,10 +88,30 @@ def measure_throughput_info(
 
 
 def measure_network_rps(config: BloomConfig) -> float:
-    proc = subprocess.run("python3 -m petals.cli.speed_test --json", shell=True, capture_output=True)
-    if proc.returncode != 0:
-        raise RuntimeError(f"Failed to measure network throughput (stdout: {proc.stdout}, stderr: {proc.stderr})")
-    network_info = json.loads(proc.stdout)
+    try:
+        import speedtest
```

Comment:
The import should be on top of the module, if possible. This is not an optional dependency, and thus the module needs to fail as early as possible

Reply:
fixed